modules/client-configuration/pages/clients-raspberrypios.adoc

```diff
@@ -0,0 +1,214 @@
+[[clients-raspberrypios]]
+= Registering {raspberrypios} Clients
+
+This section contains information about registering clients running {raspberrypios} operating systems.
+
+Bootstrapping can be used with {raspberrypios} clients for performing initial state runs, and for profile updates.
+
+ifeval::[{suma-content} == true]
+// Use list format: repository URLs / packages and metadata / version info
+[IMPORTANT]
+====
+* {raspberrypios} repository URLs are available from {scc}
+* Packages and metadata are provided by Raspberry Pi OS, not by {suse}
+* For supported products, see the support table and the release notes
+====
+endif::[]
+
+== Prepare to Register
+
+Some preparation is required before you can register {raspberrypios} clients to the {productname} Server:
+
+* Ensure DNS is correctly configured and provides an entry for the client.
+  Alternatively, you can configure the ``/etc/hosts`` files on both the {productname} Server and the client with the appropriate entries.
+* The client must have the date and time synchronized with the {productname} Server before registration.
+
+
+
+== Add Software Channels
+
+Before you register {raspberrypios} clients to your {productname} Server, you need to add the required software channels, and synchronize them.
+
+The architectures currently supported are: {arm64} and {armhf}.
+For full list of supported products and architectures, see xref:client-configuration:supported-features.adoc[].
+
+ifeval::[{uyuni-content} == true]
+
+The channels you need for this procedure are:
+
+[[raspberrypios-channels-uyuni-cli]]
+[cols="1,1,1,1,1,1,1,1", options="header"]
+.Raspberry Pi OS Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+| Contributions Channel
+| Non Free Channel
+| Non Free Firmware Channel
+| Raspberry Channel
+
+| {raspberrypios} 12 {arm64}
+| raspberrypios-12-pool-arm64-uyuni
+| raspberrypios-12-arm64-uyuni-client
+| raspberrypios-12-arm64-main-updates-uyuni
+| raspberrypios-12-arm64-contrib-uyuni
+| raspberrypios-12-arm64-non-free-uyuni
+| raspberrypios-12-arm64-non-free-firmware-uyuni
+| raspberrypios-12-arm64-raspberry-uyuni
+
+| {raspberrypios} 12 {armhf}
+| raspberrypios-12-pool-armhf-uyuni
+| raspberrypios-12-armhf-uyuni-client
+| -
+| raspberrypios-12-armhf-contrib-uyuni
+| raspberrypios-12-armhf-non-free-uyuni
+| -
+| raspberrypios-12-armhf-raspberry-uyuni
+
+|===
+
+The {arm64} architecture also requires the following channels:
```

Comment:
Looking at how this look renders, I think we maybe want  different solution:

![image](https://github.com/uyuni-project/uyuni-docs/assets/4226070/a0728d2a-d357-462a-8d8b-42baa275bbd7)

Right now users could take the channels  from the first table without 

Maybe one table for arm64 and another for armhf, both following the format of this second table.

Check with the doc squad.

Reply:
Can you try a 3-column solution? (Basically like table 1, just rows and columns switched).  Pseudo code:
```
C1 Channel descriptions | C2 {arm64}                        | C3 {armhf}
Base Channel            | raspberrypios-12-pool-arm64-uyuni | raspberrypios-12-pool-armhf-uyuni
...
```